zip-0000.rst

```diff
@@ -538,12 +605,10 @@ Draft or Withdrawn.
 
 A ZIP may only change status from Draft (or Rejected) to Proposed, when
 the Owner deems it is complete and there is rough consensus on the
-forums, validated by both the Electric Coin Company and Zcash Foundation
-Editors. One Editor will not suffice — there needs to be consensus
-among the Editors. If it's a Consensus ZIP, a Deployment section MUST
-be present in order for the ZIP to change status to Proposed. Typically,
-although not necessarily, this will specify a network upgrade in which
-the consensus change is to activate.
+forums, validated by consensus among the current ZIP Editors. If it's a
+Consensus ZIP, a Deployment section MUST be present in order for the ZIP
+to change status to Proposed. Typically, although not necessarily, this
+will specify a network upgrade in which the consensus change is to activate.
```

Comment:
I removed the specific ECC and ZF requirement here, because we wanted to apply consensus to all significant decisions, not just the Proposed status.

Should we clarify the ECC/ZF requirement where it is first mentioned?
https://github.com/zcash/zips/pull/707/files#diff-6681881651b0d57305d4bfe1eaf84d8f48ab00ec5968a1046370a6f981183085R174-R176